helm_deploy/laa-estimate-eligibility/values/ccq-uat.yaml

```diff
@@ -0,0 +1,112 @@
+# Default values for laa-check-client-qualifies-uat.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 2
+
+resources: {}
+
+image:
+  repository: nginx
+  tag: stable
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: false
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: "cd-serviceaccount"
+
+podSecurityContext: {}
+# fsGroup: 2000
+
+service:
+  type: ClusterIP
+  port: 80
+
+securityContext:
+  runAsUser: 1000
+
+ingress:
+  hosts:
+#    - host: main-check-client-qualifies-legal-aid-uat.cloud-platform.service.justice.gov.uk
+    - host: main-ccq-uat.cloud-platform.service.justice.gov.uk
+      secret: false
+  annotations:
+    external-dns.alpha.kubernetes.io/set-identifier: "check-client-qualifies-laa-estimate-eligibility-laa-check-client-qualifies-uat-green"
+    # TODO: the weight is set to 0 for now but will need to be reverted to 100 at some point
```

Comment:
just a fail safe we agreed on, when host is changed this will already be at zero.

Reply:
This was all quite trial and error - @willc-work and I had a good chat with @exonian about ingress settings. I think because we had managed to get the deployment working in this way, we stuck with it. I think 0 in this particular context might have the same affect as setting to 100? But you're right, we have changed the identifier, ingress and host so perhaps we don't need this. I will defer to Will and Michael!